### DISCUSSION TITLE
HCI usability improvement ideas

### DISCUSSION THREAD

**Discussion by ScottWilliamAnderson** [takeyamayuki/NonMouse#26] [Ideas]
Hi there, big fan of your work!

I think this is brilliant software to enable those who cannot use the mouse or trackpad due to injury or special requirements. The tracking is excellent, and the ideas to connect this with easy to use control really shows the excellence of the work! :)

I noticed that running this on a laptop, the times in which I want to activate this software, (like when cooking!) it would be great if you could activate the mouse movement with a hand gesture as well, instead of needing to be near the keyboard to push-to-activate (who doesn't hate a dirty keyboard...). Something like a ✋ to toggle the activation, a bit like this:

![High_Five_Start_Toggle](https://github.com/takeyamayuki/NonMouse/assets/47524721/ca510f77-f686-4eee-afa4-9771e394364e)

Also, one additional thing I found is that if the laptop is not placed high enough to result in the camera line being above my head, with a large screen I often run out of space in the camera's view to scroll from the top of my monitor to the bottom. It would be very helpful if the activation of the mouse movement also worked with the fingers pointing to the floor as well as the ceiling

![Downward_Scroll](https://github.com/takeyamayuki/NonMouse/assets/47524721/5291a394-dfe6-4ff6-9d17-6ab314e8dcf7)

I've used Sensitivity of 55 in Normal camera setup.
I hope you continue the great work and I look forward to perhaps one day contributing a PR!

Best,
Scott

**Comment by takeyamayuki** [comment]
Hi, @ScottWilliamAnderson 

Thanks, very good idea!
It would be good to be able to choose between key-activated and gesture-activated in the parameter adjustment window.
I am looking forward to your PR.
